api/src/main/java/liquibase/ext/sqlgenerator/core/ModifyColumnChange.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package liquibase.ext.sqlgenerator.core;
+
+import java.util.ArrayList;
+import java.util.List;
+import liquibase.change.AbstractChange;
+import liquibase.change.ChangeMetaData;
+import liquibase.change.ChangeWithColumns;
+import liquibase.change.ColumnConfig;
+import liquibase.change.DatabaseChange;
+import liquibase.change.DatabaseChangeProperty;
+import liquibase.database.Database;
+import liquibase.statement.SqlStatement;
+import liquibase.util.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
Were some of these classes just copied and pasted from here? http://svn.openmrs.org/repo/openmrs-contrib/liquibase-ext/

Reply:
Possibly, I get them via @ibacher 's pull request. 

@ibacher were the classes just copied or modified as well?

Reply:
I had a look at the package `liquibase.ext.sqpgenerator.core` in the svn repository, the classes have been refactored a bit. I did not check line by line but the original files contained code that had been commented out. These lines were removed. 

And the test coverage increased by the tests I had added.